data/speakers/alexandre_gallice.yml

```
@@ -4,8 +4,8 @@ name: Alexandre Gallice
 feature: false
 company: Red Hat
 companyLogo: /images/partners/red_hat.png
-photoUrl: https://avatars.githubusercontent.com/u/22151349?v=4
-city: Aix-en-Provence, France
+photoUrl: https://github.com/aldettinger/events/blob/main/photo/id.png
```

il faudrait rajouter `?raw=true`, sinon c'est pas directement l'image mais la page github qui donne accès à l'image.
```suggestion
photoUrl: https://github.com/aldettinger/events/blob/main/photo/id.png?raw=true
```